srcpkgs/fscrypt/template

```diff
@@ -1,14 +1,27 @@
 # Template file for 'fscrypt'
 pkgname=fscrypt
 version=0.2.6
-revision=1
-build_style=go
-go_import_path=github.com/google/fscrypt
-go_package="${go_import_path}/cmd/fscrypt"
-makedepends="pam-devel"
+revision=2
+build_style=gnu-makefile
+makedepends="go m4 pam-devel"
 short_desc="Tool for managing Linux filesystem encryption"
 maintainer="Andrea Brancaleoni <abc@pomel.me>"
 license="Apache-2.0"
 homepage="https://github.com/google/fscrypt"
 distfiles="https://github.com/google/fscrypt/archive/v${version}.tar.gz"
 checksum=bce54ebb716706150b759052665a29d01963d8df334ad9beb34105ce62d2de94
+conf_files="/etc/pam.d/fscrypt"
+
+make_install_args='PREFIX=${DESTDIR}/usr'
+make_install_target="install-pam"
+nostrip=yes
```

Comment:
This is wrong, the pam module should still be stripped. Use `nostrip_files` to list the files that shouldn't be stripped.

Reply:
Passing only the binary to `nostrip_files` fixed the issue, thanks!